nntrainer/graph/network_graph.cpp

```diff
@@ -561,7 +562,8 @@ NetworkGraph::canExecuteInPlace(const std::shared_ptr<LayerNode> &lnode) {
    */
   auto io_independent_backwarding =
     [](const std::shared_ptr<LayerNode> &lnode) {
-      return lnode->getType() == BatchNormalizationLayer::type;
+      return lnode->getType() == BatchNormalizationLayer::type ||
```

Comment:
How about adding parenthesis to clear meaning?

Reply:
Done